### PR TITLE
Rename locus columns (no tags)

### PR DESCRIPTION
**Pull request by d0choa**
```      
        "name": "is95CredibleSet",
        "name": "is99CredibleSet",
        "name": "logABF",
        "name": "posteriorProbability",
        "name": "variantId",
        "name": "pValueMantissa",
        "name": "pValueExponent",
        "name": "pValueMantissaConditioned",
        "name": "pValueExponentConditioned",
        "name": "beta",
        "name": "standardError",
        "name": "betaConditioned",
        "name": "standardErrorConditioned",
        "name": "r2Overall",
  ```

### CODE REVIEW

File: src/otg/assets/schemas/study_locus.json [hunk 1]
```diff
@@ -172,43 +172,55 @@
             },
             {
               "metadata": {},
-              "name": "tagVariantId",
+              "name": "variantId",
```
Comment (DSuveges) [inline src/otg/assets/schemas/study_locus.json:175]: I think this is the only field in the locus object that could be nullable: false.

File: src/otg/method/clump.py [hunk 2]
```diff
@@ -32,17 +32,17 @@ def _is_lead_linked(
...
             Column: Boolean in which True indicates that the lead is linked to another tag in the same dataset.
         """
         leads_in_study = f.collect_set(variant_id).over(Window.partitionBy(study_id))
         tags_in_studylocus = f.array_union(
             # Get all tag variants from the credible set per studyLocusId
-            f.transform(credible_set, lambda x: x.tagVariantId),
+            f.transform(ld_set, lambda x: x.tagVariantId),
             # And append the lead variant so that the intersection is the same for all studyLocusIds in a study
-            f.array(f.col("variantId")),
+            f.array(variant_id),
```
Comment (DSuveges) [inline src/otg/method/clump.py:45]: How did it work in the original implementation when it was:
```python
f.array(f.col("variantId")),
```
however `variant_id` was a column object already?

Comment (d0choa) [reply src/otg/method/clump.py:45]: It was working, but the column name was hardcoded. The function parameter was ignored. @ireneisdoomed spotted this one

**Comment by DSuveges** [comment]
Interestingly some tests were failing:
```
FAILED tests/method/test_pics.py::TestFinemap::test_finemap_null_r2 - py4j.protocol.Py4JJavaError: An error occurred while calling o9989.showString.
FAILED tests/method/test_pics.py::TestFinemap::test_finemap_empty_array - py4j.protocol.Py4JJavaError: An error occurred while calling o9747.showString.

Results (102.38s (0:01:42)):
     141 passed
       2 failed
         - tests/method/test_pics.py:46 TestFinemap.test_finemap_null_r2
         - tests/method/test_pics.py:24 TestFinemap.test_finemap_empty_array
```

However, when re-running with the problematic test file only, all passed:
```bash
╰─ poetry run pytest --doctest-modules tests/method/test_pics.py
```

```
Results (43.42s):
       5 passed
```

**Review by DSuveges — APPROVED** [review]
Assuming the `tag` prefix only stays in `study_locus.ldSet.tagVariantId`, `study_locus_overlap.tagVariantId` and `ld_index.ldSet.tagVariantId` (or change later). Things look OK. Tests are passing. I haven't seen accidentally left refereces. It looks good.

**Comment by d0choa** [comment]
yes, let's discuss what we want to do with `ldSet`.

**Pull request merged**